bank.rb

```diff
@@ -152,8 +152,8 @@ def self.owner_account
       sample = CSV.read("./support/account_owners.csv")
       owner_accounts_array =[]
```

Comment:
When you submit code, i'd delete the commented out bits that you were still debugging or working through